pyqmc/dmc.py

```diff
@@ -141,6 +143,7 @@ def dmc_propagate(wf,configs,weights,tstep,nsteps=5,accumulators=None,ekey=('ene
 
     eloc = accumulators[ekey[0]](configs, wf)[ekey[1]]
     eref_mean = np.mean(weights*eloc)/np.mean(weights)
+    eref_sigma = np.mean(weights*eloc/np.mean(weights))
```

Comment:
This will result in a different cutoff at every propagate. Also consider that if this is run in parallel, sigma might only be evaluated on a few configurations, and it will be different for every node. dmc_propagate should just get passed branchcut_start and branchcut_stop in energy units.